modules/p2/ding_list/theme/theme.inc

```diff
@@ -164,24 +165,33 @@ function theme_ding_list_lists($variables) {
       }
     }
 
-    $out .= theme('item_list', array(
-      'items' => $rows,
-      'title' => $group['title'],
-      'type' => 'ul',
-      'attributes' => array(),
-    ));
+    if (!empty($rows)) {
+      $render = array(
+        '#theme' => 'item_list',
+        '#title' => $group['title'],
+        '#items' => $rows,
+        '#title' => $group['title'],
+        '#type' => 'ul',
+        '#attributes' => array(),
+      );
+      $out .= drupal_render($render);
```

Comment:
Calling drupal_render is basically the same as theme().

You theme functions should return a render array and let the theme layer do the rendering. So the front-end has a change to alter in style and content.